cms/templates/js/course-outline.underscore

```diff
@@ -57,11 +83,17 @@ if (staffOnlyMessage) {
         addStatusMessage(messageType, messageText);
     }
 
-    if (hasPartitionGroups) {
+    if (hasPartitionGroups && allowedGroups.length > 1) {
         addStatusMessage(
             'partition-groups',
             gettext('Access to some content in this unit is restricted to specific groups of learners')
         );
+    } else if (allowedGroups.length === 1 && xblockInfo.isVertical()) {
+        var allowedId = allowedGroups[0];
+        var groupName = groupIdToName[allowedId];
+        messageType = 'partition-groups';
+        messageText = gettext('Access to this unit is restricted to ' + groupName + ' learners');
```

Comment:
I think there's a problem here with how gettext works with concatenation, we'll need to do something similar to http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/internationalization/i18n.html#id12 other than that looks good to me!